### PR TITLE
Add Carrier Cloudification engage page

### PR DESCRIPTION
**Pull request by MeltySnow**
## Done

- created a u.c version of https://app-sjg.marketo.com/#LP5612A1LA1

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/telco-cloudification-ebook
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page has the same content as marketo
- See that the form goes to the correct thank you page

### CODE REVIEW

File: templates/engage/telco-cloudification-ebook.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <p>Download this eBook&nbsp;to learn:</p>
+        <ul class="p-list">
+            <li class="p-list_item is-ticked">Why telecom service providers must reposition themselves beyond their traditional core business<br /></li>
+            <li class="p-list_item is-ticked">Which six key factors are responsible for disrupting the Telecoms industry<br /></li>
+            <li class="p-list_item is-ticked">How to identify the building blocks for moving towards telecom cloudification<br /></li>
+            <li class="p-list_item is-ticked">How Canonical is helping Deutsche Telekom, Spectrum (formerly Time Warner Cable) and BSkyB increase revenues, reduce operational costs and improve operational efficiencies</li>
+        </ul>
+        <p><span style="color:#E95420;font-size:18px;font-weight:bold"></span><span>Discover why Ubuntu OpenStack enables Telcos to maximise margins, reduce the costs and risks of introducing new value-added services and speed up time-to-revenue.</font></span><span style="color:#E95420;font-size:18px;font-weight:bold"></span></p>
```
Comment (pmahnke) [inline templates/engage/telco-cloudification-ebook.html]:
```suggestion
        <blockquote class="p-pull-quote">
  <p class="p-pull-quote__quote">Discover why Ubuntu OpenStack enables Telcos to maximise margins, reduce the costs and risks of introducing new value-added services and speed up time-to-revenue.</p>
  </blockquote>
```

File: templates/engage/telco-cloudification-ebook.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+        <br/>
```
Comment (pmahnke) [inline templates/engage/telco-cloudification-ebook.html]:
```suggestion

```

File: templates/engage/telco-cloudification-ebook.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        <br/>
```
Comment (pmahnke) [inline templates/engage/telco-cloudification-ebook.html]:
```suggestion

```

File: templates/engage/telco-cloudification-ebook.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+        <p><center>INCLUDES CASE STUDIES FROM</center></p>
```
Comment (pmahnke) [inline templates/engage/telco-cloudification-ebook.html]:
```suggestion
        <h5 class="p-muted-heading u-align--center">INCLUDES CASE STUDIES FROM</h5>
```

File: templates/engage/telco-cloudification-ebook.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        <table cellpadding="10" cellspacing="10" border="0">
```
Comment (pmahnke) [inline templates/engage/telco-cloudification-ebook.html]:
```<ul class="p-inline-images">
      <li class="p-inline-images__item">
        <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/1ccb6d93-logo-deutschetelekom.svg" width="144" alt="Deutshe Telekom">
      </li>
      <li class="p-inline-images__item">
        <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/be345c7f-bSkyb_logo.png?w=141" width="141" alt="Sky">
      </li>
      <li class="p-inline-images__item">
        <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/3ec677f3-spectrum_logo.png" alt="Spectrum logo" width="250" />
      </li>
</ul>```

Comment (pmahnke) [reply templates/engage/telco-cloudification-ebook.html]: you need to remove the table

Comment (MeltySnow) [reply templates/engage/telco-cloudification-ebook.html]: I've submitted a pull request with your correction.

**Pull request merged**